### ISSUE TITLE
UnlockDialog:  Crash

### ISSUE THREAD

**Issue by HashEngineering** [HashEngineering/dash-wallet#110]
There have been 20 crashes in the past week:

```
java.lang.NullPointerException: 
  at android.app.DialogFragment.dismissInternal (DialogFragment.java:296)
  at android.app.DialogFragment.dismissAllowingStateLoss (DialogFragment.java:277)
  at de.schildbach.wallet.ui.UnlockWalletDialogFragment$1.onSuccess (UnlockWalletDialogFragment.java:61)
  at de.schildbach.wallet.ui.CheckWalletPasswordTask$1$1.run (CheckWalletPasswordTask.java:44)
  at android.os.Handler.handleCallback (Handler.java:746)
  at android.os.Handler.dispatchMessage (Handler.java:95)
  at android.os.Looper.loop (Looper.java:148)
  at android.app.ActivityThread.main (ActivityThread.java:5443)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:728)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:618)
```

**Comment by HashEngineering** [comment]
@sambarboza could this be fixed by the concept of #109 ?  Is it the same issue as #104?

**Comment by sambarboza** [comment]
@HashEngineering it's a different issue. I'm investigating this.
